bigquery/indexer.py

```diff
@@ -54,33 +49,27 @@ def parse_args():
     return parser.parse_args()
 
 
-def index_facet_field(es, index_name, primary_key, project_id, dataset_id,
-                      table_name, field_name, readable_field_name,
-                      billing_project_id):
+def index_table(es, index_name, primary_key, table_name, billing_project_id):
     """Indexes a facet field.
 
-  I couldn't find an easy way to import BigQuery -> Elasticsearch. So instead:
-
-  - BigQuery -> pandas dataframe
-  - Convert datafrom to dict
-  - dict -> Elasticsearch
-
-  Args:
-    es: Elasticsearch object.
-    index_name: Name of Elasticsearch index.
-    primary_key: Name of primary key field.
-    project_id: BigQuery project ID.
-    dataset_id: BigQuery dataset ID.
-    table_name: BigQuery table name.
-    field_name: BigQuery field name.
-    readable_field_name: Field name for index and Data Explorer UI
-    billing_project_id: GCP project ID to bill
-  """
+    Args:
+        es: Elasticsearch object.
```

Comment:
In general, I prefer full variable names (e.g. elasticsearch) to the abbreviations for readability, even if Elasticsearch documentation uses shorthand.

Reply:
I do too in general, but this is a common convention with Elasticsearch code.